catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2319,9 +2319,12 @@ protected ResourceResponse getResource(ResourceRequest resourceRequest, boolean
 
         } catch (RuntimeException e) {
             LOGGER.error("RuntimeException caused by: ", e);
+            if (StringUtils.contains(e.getMessage(), "data usage limit exceeded")) {
```

Comment:
Why not just explicitly catch `DataUsageLimitExceededException` first, then fall through to the more generic `RuntimeException`?